stdlib/2.7/__builtin__.pyi

```diff
@@ -529,7 +529,7 @@ class dict(MutableMapping[_KT, _VT], Generic[_KT, _VT]):
     @overload
     def __init__(self, map: Mapping[_KT, _VT], **kwargs: _VT) -> None: ...
     @overload
-    def __init__(self, iterable: Iterable[Tuple[_KT, _VT]], **kwargs: _VT) -> None: ...
+    def __init__(self, iterable: Iterable[Union[Tuple[_KT, _VT], Sequence[_VT]]], **kwargs: _VT) -> None: ...
```

Comment:
This doesn't seem quite right. The first item of the sequence should have `_KT` as its type, not `_VT`. For example, consider this mypy example (not verified, but I believe I can predict how this would work):

```
d = dict([['x', 'y']])  # type: Dict[int, str]
```

There would be no error, since `_VT` doesn't get inferred from the argument, but clearly the type annotation is wrong and the correct type is `Dict[str, str]`. However, there is no way to represent the fact using PEP 484 features. Also, as tuple is also a sequence type inference gets tricky and a little ambiguous.